diff --git a/bin/cloudtoken b/bin/cloudtoken
index 7745775..6e36697 100755
--- a/bin/cloudtoken
+++ b/bin/cloudtoken
@@ -8,9 +8,7 @@
 CLOUDTOKEN_DIR=~/.config/cloudtoken
 DAEMON_MODE=0
 
-SOURCED_FILE=$(command which cloudtoken)
-
-INSTALL_DIR="$( cd "$( dirname "${SOURCED_FILE}" )" && pwd )"
+INSTALL_DIR=@OUT@/bin
 
 PARAMS="${@}"
 
@@ -49,4 +47,4 @@ unset INSTALL_DIR
 unset DAEMON_MODE
 unset SOURCED_FILE
 unset INSTALL_DIR
-unset PARAMS
\ No newline at end of file
+unset PARAMS
diff --git a/shell_additions/bashrc_additions b/shell_additions/bashrc_additions
index 110dd59..6ea2c13 100644
--- a/shell_additions/bashrc_additions
+++ b/shell_additions/bashrc_additions
@@ -6,7 +6,7 @@
 
 CLOUDTOKEN_DIR=~/.config/cloudtoken
 
-alias cloudtoken=". cloudtoken"
+alias cloudtoken=". @OUT@/bin/cloudtoken"
 
 unsetcloudtoken () {
     unset AWS_ACCESS_KEY_ID
